src/LibraryService.DataLayer/Models/DbBook.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.LibraryService.DataLayer.Models
+{
+  public class DbBook
+  {
+    public const string TableName = "Books";
+
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public Guid? AuthorId { get; set; }
```

Comment:
AuthorId is nullable?

Reply:
If you discussed it on the meetings, please, write comments. Like "Discussed on daily meetup"

For now, I think you need to change this.